docs/spec-conformance.md

```diff
@@ -11,8 +11,8 @@ v1.0.0       | `SCMP_ARCH_PARISC`                       | Unplanned, due to lack
 v1.0.0       | `SCMP_ARCH_PARISC64`                     | Unplanned, due to lack of users
 v1.0.2       | `.linux.personality`                     | [#3126](https://github.com/opencontainers/runc/pull/3126)
 v1.1.0-rc.1  | `.linux.resources.cpu.burst`             | [#3749](https://github.com/opencontainers/runc/pull/3749)
-v1.1.0-rc.1  | `.[]mounts.uidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
-v1.1.0-rc.1  | `.[]mounts.gidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
+v1.1.0-rc.1  | `.[]mounts.uidMappings`                  | Yes, but requires using userns with identical uidMappings ([#3717](https://github.com/opencontainers/runc/pull/3717))
```

Comment:
The PR #3717 is now merged, so the link to #3717 should be removed now.
This limitation should be probably moved to a separate table or a bullet point list.

Reply:
Fixed it, thanks!